src/app/modals/about-modal.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Component,
+  DoCheck,
+  Input,
+  Output,
+  EventEmitter,
+  TemplateRef,
+  ViewEncapsulation,
+  OnInit
+} from '@angular/core';
+
+import { cloneDeep, defaults, isEqual } from 'lodash';
+
+import { AboutModalConfig } from './about-modal-config';
+
+/**
+ * Component for rendering AboutModal
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-about-modal',
+  templateUrl: './about-modal.component.html'
+})
+
+export class AboutModalComponent implements DoCheck, OnInit {
+
+  /**
+   * The AboutModal config contaning component properties
+   */
+  @Input() config: AboutModalConfig;
+
+  /**
+   * Modal Template for custom content
+   */
+  @Input() template: TemplateRef<any>;
```

Comment:
We should probably remove the template and just use ng-content for the body of the modal -- it would be a little cleaner.